features/zoek-met-bsn.feature

```diff
@@ -0,0 +1,83 @@
+
+        Gegeven het systeem heeft een persoon met de volgende gegevens
+        | burgerservicenummer |
+        | 999999321           |
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                     |
+        | type                | ZoekMetBurgerservicenummer |
+        | burgerservicenummer | 999999321                  |
+        | fields              | burgerservicenummer        |
+        Dan bevat de response alleen personen met de volgende gegevens 
+        | burgerservicenummer |
+        | 999999321           |
+
+    Scenario: Zoek met meerdere valide burgerservicenummers
+
+        Gegeven het systeem heeft een persoon met de volgende gegevens
+        | burgerservicenummer |
+        | 999999321           |
+        En het systeem heeft een persoon met de volgende gegevens
+        | burgerservicenummer |
+        | 999995492           |
+        En het systeem heeft een persoon met de volgende gegevens
+        | burgerservicenummer |
+        | 999991802           |
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                        |
+        | type                | ZoekMetBurgerservicenummer    |
+        | burgerservicenummer | 999999321,999995492,999991802 |
+        | fields              | burgerservicenummer           |
+        Dan bevat de response alleen personen met de volgende gegevens 
+        | burgerservicenummer |
+        | 999999321           |
+        | 999995492           |
+        | 999991802           |
+
+    @fout-case
+    Scenario: Zoek zonder burgerservicenummers
+        Als personen wordt gezocht met de volgende parameters
+        | naam   | waarde                     |
+        | type   | ZoekMetBurgerservicenummer |
+        | fields | burgerservicenummer        |
+        Dan bevat de response een problemDetails met de volgende velden
+        | naam   | waarde                                        |
+        | title  | Één of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+
+    @fout-case
+    Abstract Scenario: <titel>
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                     |
+        | type                | ZoekMetBurgerservicenummer |
+        | burgerservicenummer | <burgerservicenummers>     |
+        | fields              | burgerservicenummer        |
+        Dan bevat de response een problemDetails met de volgende velden
+        | naam   | waarde                                        |
+        | title  | Één of meerdere parameters zijn niet correct. |
+        | status | 400                                           |
+
+        Voorbeelden:
+        | burgerservicenummers | titel                                   |
+        |                      | Zoek met leeg burgerservicenummer lijst |
```

Comment:
dit is denk ik hetzelfde als het vorige scenario (Zoek zonder burgerservicenummers)?

Reply:
is niet hetzelfde. Bij vorige scenario ziet de requestbody er als volgt uit:

{
    "type": "ZoekMetBurgerservicenummer",
    "fields": "burgerservicenummer"
}
en bij deze scenario als volgt
{
    "type": "ZoekMetBurgerservicenummer",
    "fields": "burgerservicenummer",
    "burgerservicenummer": []
}
Vorige scenario kan worden gezien als een 'technische' validatie omdat die niet mogelijk is als de client code wordt gegenereerd uit de OAS spec. Maar achterliggende automation code laat wel zien dat deze situatie kan voorkomen als er geen code generatie wordt gebruikt. Ik vind het prima om de vorige scenario weg te halen als jullie het niet relevant vinden.